daemon/container_unix.go

```diff
@@ -976,9 +987,9 @@ func (container *Container) connectToNetwork(idOrName string, updateSettings boo
 		if err != nil {
 			return err
 		}
-	}
 
-	container.updateSandboxNetworkSettings(sb)
+		container.updateSandboxNetworkSettings(sb)
```

Comment:
ok apparently it doesn't return anything.
I'm also not sure why this was moved inside the if.
If WalkSandboxes sets `sb` to something else than `nil`, then in the previous code we call updateSandboxNetworkSettings with that `sb`, whereas in your current PR, we wouldn't call it.